rules/providers.bzl

```diff
@@ -1,6 +1,11 @@
 FrameworkInfo = provider(
     fields = {
         "vfsoverlay_infos": "Merged VFS overlay infos, present when virtual frameworks enabled",
-        "framework_headers": "Headers part of the framework's public interface",
```

Comment:
So this is a breaking API change right? Just checking to ensure this is intentional

Reply:
Yes, FrameworkInfo was just introduced to support [vfs framework](https://github.com/bazel-ios/rules_ios/pull/276) by @jerrymarino. I believe it's only used internally.

Reply:
@jerrymarino PTAL